.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
+title: ''
+labels: ''
+assignees: ''
+
```

Comment:
[![Confirm internal staff meant to post in public](https://github.com/github/docs/actions/workflows/confirm-internal-staff-work-in-docs.yml/badge.svg)](https://github.com/github/docs/actions/workflows/confirm-internal-staff-work-in-docs.yml)